modules/getting-started-web-console-login.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-web-console-login_{context}"]
+= Logging in to the web console
+
+You can log in to the {product-title} web console to access and manage your cluster.
+
+.Prerequisites
+
+* {product-title} web console login credentials.
```

Comment:
Suggest making this a full sentence. Like for the CLI one, you said "You must have access to an {product-title} cluster."